src/plugins/discourse/createGraph.test.js

```diff
@@ -44,13 +44,13 @@ describe("plugins/discourse/createGraph", () => {
     posts(): $ReadOnlyArray<Post> {
       return this._posts;
     }
-    users(): $ReadOnlyArray<string> {
-      const users = new Set();
+    users(): $ReadOnlyArray<User> {
+      const users = [];
```

Comment:
This array will now have two copies of `{username: "decentralion", trustLevel: 3}`, which is not what we want.

Reply:
We could fix this by having a set track the usernames, and only adding the user to the user array if their username is not yet in the set. However, I think that's too complicated: instead, let's just have this method explicitly return the right set of users that are included in the test case. That way we can also give the users some varying trust levels, which will be good for an example test case later once we actually use the trust levels.